pages/docs/mutation.en-US.md

```diff
@@ -1,16 +1,24 @@
 # Mutation
 
+SWR provides the `mutate` and `useSWRMutation` APIs to mutate remote data and its cache.
+
+## mutate
+
 ```js
 mutate(key, data, options)
 ```
 
-## Options
+### API
 
-- `optimisticData`: data to immediately update the client cache, or a function that receives current data and returns the new client cache data, usually used in optimistic UI.
-- `revalidate`: should the cache revalidate once the asynchronous update resolves.
-- `populateCache`: should the result of the remote mutation be written to the cache.
-- `populateCache`: should the result of the remote mutation be written to the cache, or a function that receives new result and current result as arguments and returns the mutation result.
-- `rollbackOnError`: should the cache rollback if the remote mutation errors.
+#### Parameters
+
+- `key`: same as `useSWR`'s `key`
+- `data`: data to update the client cache, or an async function for the remote mutation
+- `options`: accepts the following options
+  - `optimisticData(currentData)`: data to immediately update the client cache, or a function that receives current data and returns the new client cache data, usually used in optimistic UI.
+  - `revalidate = true`: should the cache revalidate once the asynchronous update resolves.
```

Comment:
We can still keep the name here instead of showing an assignment expression, and give the default value in the description
```suggestion
  - `revalidate`: should the cache revalidate once the asynchronous update resolves (default is `true`).
```

Reply:
@huozhi 
The API documentation of `useSWR` uses this assignment expression style, so I've used the same style because of the consistency.
https://swr.vercel.app/docs/options#options

Should we use this style in other places?